src/main/java/com/cleancommunity/user/UserController.java

```diff
@@ -9,15 +12,15 @@
 @RestController
 public class UserController {
 
-//    private UserDAO userDAO = new UserDAO();
-//    private UserFactory userFactory = new UserFactory();
+//    private com.cleancommunity.com.cleancommunity.user.UserDAO userDAO = new com.cleancommunity.com.cleancommunity.user.UserDAO();
```

Comment:
Why is this so long lol. Since they're all in the same package, is this necessary?